searx/templates/__common__/about.html

```diff
@@ -48,13 +47,13 @@ <h2 id="dev_faq">Developer FAQ</h2>
 
 <h3>New engines?</h3>
 <ul>
-    <li>Edit your <a href="https://raw.github.com/asciimoo/searx/master/searx/settings.yml">settings.yml</a></li>
```

Comment:
May be wrong: 
* here it is about searx.
* bellow ```{{ brand.DOCS_URL }}/admin/installation.html#uwsgi``` is about the fork.
* I've just suggested to make a link to the fork git repository instead of the searx git repository.

I think it can be confusing.

Reply:
> [EDIT] I'm not sure to know what is the right solution.

me too :) .. my thought is, that settings.yml and 'custom engine' are just examples showing how it looks like.  Those should link to searx .. 

This "Installation/WSGI support" should be dropped .. this is nothing one want see on a about page.

What do you think?

Reply:
May we should also drop "New engines?" and "How to debug" ..

Reply:
> EDIT2] Another idea: two pages, one only searx, another one about the fork if it is fork.

I am not happy with .. lets drop all what is not in the scope of a common user .

An "about" page should only address higher-level issues that are important to each user. No developer aspects.